src/status_im2/contexts/chat/photo_selector/view.cljs

```diff
@@ -80,31 +81,38 @@
       (inc (utils/first-index #(= (:uri item) (:uri %)) @selected))])])
 
 (defn album-title
-  [photos? selected-album selected temporary-selected]
-  [rn/touchable-opacity
-   {:style               (style/title-container)
-    :active-opacity      1
-    :accessibility-label :album-title
-    :on-press            (fn []
-                           (if photos?
-                             (do
-                               (reset! temporary-selected @selected)
-                               (rf/dispatch [:open-modal :album-selector]))
-                             (rf/dispatch [:navigate-back])))}
-   [quo/text
-    {:weight          :medium
-     :ellipsize-mode  :tail
-     :number-of-lines 1
-     :style           {:max-width 150}}
-    selected-album]
-   [rn/view {:style (style/chevron-container)}
-    [quo/icon (if photos? :i/chevron-down :i/chevron-up)
-     {:color (colors/theme-colors colors/neutral-100 colors/white)}]]])
+  [photos? selected-album]
```

Comment:
this looks suspicious, why you don't want to react on params changes ?

Reply:
I commented out the function, so had to remove the params for `lint`. Currently, album selector is disbaled (due to performance issues: https://github.com/status-im/status-mobile/issues), but already fixed. I did not want to mix it with this PR.